cobalt/media/media_module.cc

```diff
@@ -198,6 +198,9 @@ bool MediaModule::SetConfiguration(const std::string& name, int32 value) {
     LOG(INFO) << (value ? "Enabling" : "Disabling")
               << " media metrics collection.";
     return true;
+  } else if (name == "MaxInputSize") {
```

Comment:
Max input size is per stream, instead of per video.  Imagine that we are playing a 4k video with a thumbnail video at the same time, we'd like to increase the max input size for the 4k video, but not the thumbnail video.
https://github.com/youtube/cobalt/pull/1681 is an example on adding per stream attribute.

Reply:
Moved to HTMLVideoElement.